src/test/groovy/graphql/normalized/ExecutableNormalizedOperationFactoryTest.groovy

```diff
@@ -198,8 +198,8 @@ type Dog implements Animal{
 
         Document document = TestUtil.parseQuery(query)
 
-        ExecutableNormalizedOperationFactory dependencyGraph = new ExecutableNormalizedOperationFactory()
-        def tree = dependencyGraph.createExecutableNormalizedOperation(graphQLSchema, document, null, CoercedVariables.emptyVariables())
```

Comment:
No point in creating an instance of `ExecutableNormalizedOperationFactory`, since the API is all static methods